src/brain/issueLabelHandler/index.test.ts

```diff
@@ -691,6 +691,41 @@ describe('issueLabelHandler', function () {
       );
     });
 
+    it('should remove `Waiting for: Product Owner` label when staff member comments and issue is waiting for product owner', async function () {
```

Comment:
nit: "should remove `Waiting for: Product Owner` label when staff member comments and issue already has the `Waiting for: Product Owner` label"

Or am I misunderstanding this test, and `is waiting for product owner` refers to a condition other than already having that label?

Reply:
you're understanding this correctly, I will change for more clarity